packages/app/src/pages/index.tsx

```diff
@@ -17,15 +17,39 @@
  * along with Essencium Frontend. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Home } from '@frachtwerk/essencium-lib'
+import { FeedBackWidget, Home } from '@frachtwerk/essencium-lib'
```

Comment:
Should we use `FeedbackWidget` or `FeedBackWidget`? Since feedback is one word I'd suggest `FeedbackWidget`.

Reply:
Yes, you are right. FeedbackWidget is the better choice.